bundler/helpers/v2/spec/functions_spec.rb

```diff
@@ -5,9 +5,7 @@
 RSpec.describe Functions do
   # Verify v1 method signatures are exist, but raise as NYI
   {
-    private_registry_versions: [:gemfile_name, :dependency_name, :dir, :credentials ],
-    jfrog_source: [:dir, :gemfile_name, :credentials, :using_bundler2],
-    git_specs: [:dir, :gemfile_name, :credentials, :using_bundler2],
+    jfrog_source: %i(dir gemfile_name credentials using_bundler2)
```

Comment:
I wonder if this function isn't tested anywhere?

Reply:
😬  Possible, we should have cleared out this file by now, right?

Reply:
I don't think it is, I'll add some coverage 👍